src/main/java/Duke.java

```diff
@@ -1,10 +1,95 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+import java.io.IOException;
+import java.util.*;
+import java.io.File;
+
+
+
+public class Duke{
+    public static void main(String[] args) throws IOException,Exception {
+        Scanner sc = new Scanner(System.in);
+        String mesInput = "";
+        ArrayList<Task> arrTask = new ArrayList<Task>();
+//        TaskList tList = new TaskList(arrTask);
+        Ui uiDisplay = new Ui();
+        Storage  fileStorage = new Storage();
+        String filePath = "duke.txt";
+        File f = new File(filePath);
+
+        while(!mesInput.equalsIgnoreCase("bye")){
+            mesInput = sc.nextLine();
+            String [] s = new String[2];
+            Parser userCommand = new Parser(mesInput,uiDisplay,fileStorage);
+            if(mesInput.equalsIgnoreCase("bye")){
+                break;
+            }
+
+            if(userCommand.getCommandType(mesInput).equals("todo") && !mesInput.equalsIgnoreCase("todo")){
```

Comment:
I see what you're trying to do here with the if-else statements (to handle incorrect commands) instead of a switch-case control structure, but perhaps it could be coded in a more uniform and hence clearer way?